tools/tracer/README

```diff
@@ -0,0 +1,60 @@
+THIS DOCUMENT CONTAINS INFORMATION ON PRODUCTS IN THE DESIGN PHASE OF DEVELOPMENT.
+
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
+
+
+Overview
+========
+
+Media SDK Tracer is a tool which permits to dump logging information from the calls
+of the application to the Media SDK library. Trace log obtained from this tool is a
+recommended information to provide to Media SDK team on submitting questions and
+issues.
+
+Media SDK tracer tool version is: v1.32.0. Note that 1.32 stands for the Media SDK API
```

Comment:
Remove mentioning of specific versions. Remove the whole paraghraph, please.

Reply:
done